lib/ph_model/version.rb

```diff
@@ -0,0 +1,3 @@
+module PhModel
+  VERSION = '0.0.1'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.